### DISCUSSION TITLE
Clicking “New discussion” when in a category should automatically place that discussion into that category

### DISCUSSION THREAD

**Discussion by hughlilly** [community/community#6169] [Discussions]
When I have selected a category in the sidebar (such as “Discussions” here), and I click “New discussion”, I would like for the discussion I am creating to be automatically in that category.

Surely if one is browsing discussions in a given category and clicks “New discussion”, there is a high likelihood that they want to contribute a discussion to that category more than to any other. In my opinion, it’s simple enough to change the category before (and even after) posting, and that having something selected, rather than nothing, is good because it takes a bit of work away from the user. Perhaps my suggestion breaks some fundamental law of user-software dynamics that I’m not aware of, but I don’t believe the majority of users would always want to be prompted to select a category.

Thanks,
Hugh

**Comment by laymonage** [comment]
Related: #5640.

**Reply by hughlilly** [reply]
Thanks; that's laid out in a much more structured way—perhaps I should have done the same.
